src/Microsoft.Health.Fhir.Liquid.Converter/Hl7v2/Hl7v2DataParser.cs

```diff
@@ -42,7 +42,10 @@ public static Hl7v2Data Parse(string message)
                     result.Data.Add(segment);
                 }
 
-                return result;
+                return new Dictionary<string, object>
+                {
+                    { "hl7v2Data", result },
```

Comment:
Since the key `"hl7v2Data"` is exclusive(currently) and widely-used, how about making it a public and static member of the `Hl7v2DataParser` class? This can also avoid misspelling.

Reply:
Thanks! It is added in the update.